test/factories/noises.rb

```diff
@@ -32,7 +32,7 @@
     text { Faker::Lorem.sentences.join(' ') }
     created_at { Time.now }
 
-    Neighborhood::DISTRICTS.each do |key, district|
+    Neighborhood::districts.each do |key, district|
```

Comment:
Unused block argument - `district`. If it's necessary, use `_` or `_district` as an argument name to indicate that it won't be used.<br>Do not use `::` for method calls.